### ISSUE TITLE
Parenthesis after "line-long" whitespace results in "line length was zero"

### ISSUE THREAD

**Issue by mzellho** [davidmoten/word-wrap#105]
Hi @davidmoten,

you thought about so many things in your library and provided a nice API, making it a truly great library to work with, thanks a lot for sharing it with us. I should have said that earlier and without raising an issue simultaneously.

Unfortunately, I ran into a small problem this morning: If the text to be wrapped starts with an opening parenthesis after at least maxWidth spaces, we get a "line length was zero". I prepared a little test case here: https://github.com/mzellho/word-wrap/commit/36bdb6feaed801c69cf52576395e3028e69c774f.

Does this description make sense?

Best regards,
Max

**Comment by davidmoten** [comment]
Glad the library is useful. Thanks for the report, I'll look shortly.

**Comment by davidmoten** [comment]
Just looking at this now. What result do you want (obviously we don't want an exception thrown)? If I remove the precondition check then the whitespace is gobbled up:

```java
assertEquals(
    "(abc", 
    WordWrap.from(repeat(" ", maxWidth) + "(abc") //
        .maxWidth(maxWidth)
        .wrap());
```

**Comment by davidmoten** [comment]
Fix released in 0.1.13 on Maven Central now.
